### PR TITLE
Merge changes from 19.4 final version

### PR DESCRIPTION
**Pull request by mokagio**
Notice that there are a bunch of localization related commits after the version bump. That's because some of the new translations weren't compliant with the App Store Connect limitations. I fixed them manually and locally to get the metadata upload to ASC done ASAP. I'm going to flag them with the localization team afterwards.

### CODE REVIEW

File: fastlane/metadata/fr-FR/subtitle.txt [hunk 1]
```diff
@@ -1 +1 @@
-Concevez un site, créez un blog
\ No newline at end of file
+Concevez un site créez un blog
```
Comment (mokagio) [inline fastlane/metadata/fr-FR/subtitle.txt]: This is a brilliant fix for the 1 extra character issue with the `fr-FR` translation! I wish I thought of that 🤦‍♂️

File: fastlane/metadata/he/keywords.txt [hunk 2]
```diff
@@ -1 +1 @@
-רשתות חברתיות,פתקים,Jetpack,כתיבה,תיוג גיאוגרפי,מדיה,בלוג,אתר,כתיבת בלוג,כתיבת יומן
\ No newline at end of file
+blogger,writing,blogging,web,maker,online,store,business,make,create,write,blogs
```
Comment (mokagio) [inline fastlane/metadata/he/keywords.txt]: 🤔 

Looks like the GlotPress translation was... English? 

image

**Review by mokagio — COMMENTED** [review]
- [x] Fastlane  localized `release_notes.txt` updates
- [ ] ~~Localized `.strings` updates~~ N.A. Clearly they all came through already in the previous beta
- [x] Build number bump in version `.xcconfig` file

File: fastlane/jetpack_metadata/he/release_notes.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Color us excited—support for multiple color palettes is here! Add Theme and Default color and gradient palettes, or create your own Custom palette with your favorite shades. We also fixed an issue that caused theme colors not to appear and current theme data not to refresh.
+
+For devices using right-to-left languages, we updated icon, button, and text positioning for the “Choose a Domain” screen in the site creation process. Alright, alright.
+
+Want to see rich media like images and .gifs in your device’s push notifications? Just press and hold the notification to open a preview.
+
+We made some tweaks to Weekly Roundup notifications to make sure they’re showing up once you’ve enabled them. Giddyup!
+
+Finally, we added a few fixes on the Insight screen that’ll help speed things up. We also squashed a bug that bolded auto-corrected words in content block headings.
```
Comment (AliSoftware) [inline fastlane/jetpack_metadata/he/release_notes.txt:9]: English text for hebrew locale?

File: fastlane/metadata/ar-SA/keywords.txt [hunk 4]
```diff
@@ -1 +1 @@
-مدون، كتابة، تدوين، ويب، صانع، عبر الإنترنت، متجر، أعمال، إعداد، إنشاء، كتابة، مدونات
\ No newline at end of file
+مدون, كتابة, تدوين, ويب, صانع, عبر الإنترنت, متجر, أعمال, إعداد, إنشاء, كتابة, مدونات
```
Comment (AliSoftware) [inline fastlane/metadata/ar-SA/keywords.txt]: Nice catch 👌

File: fastlane/metadata/he/keywords.txt (hunk 2 above)
Comment (AliSoftware) [inline fastlane/metadata/he/keywords.txt]: English copy for Hebrew locale 🤔

File: fastlane/metadata/he/name.txt [hunk 5]
```diff
@@ -0,0 +1 @@
+WordPress – Website Builder
```
Comment (AliSoftware) [inline fastlane/metadata/he/name.txt:1]: English copy for Hebrew locale

File: fastlane/metadata/he/release_notes.txt [hunk 6]
```diff
@@ -0,0 +1,9 @@
+Color us excited—support for multiple color palettes is here! Add Theme and Default color and gradient palettes, or create your own Custom palette with your favorite shades. We also fixed an issue that caused theme colors not to appear and current theme data not to refresh.
+
+For devices using right-to-left languages, we updated icon, button, and text positioning for the “Choose a Domain” screen in the site creation process. Alright, alright.
+
+Want to see rich media like images and .gifs in your device’s push notifications? Just press and hold the notification to open a preview.
+
+We made some tweaks to Weekly Roundup notifications to make sure they’re showing up once you’ve enabled them. Giddyup!
+
+Finally, we added a few fixes on the Insight screen that’ll help speed things up. We also squashed a bug that bolded auto-corrected words in content block headings.
```
Comment (AliSoftware) [inline fastlane/metadata/he/release_notes.txt:9]: English copy for Hebrew locale

File: fastlane/metadata/he/subtitle.txt [hunk 7]
```diff
@@ -1 +1 @@
-לבנות אתר או בלוג
\ No newline at end of file
+Design a site, build a blog
```
Comment (AliSoftware) [inline fastlane/metadata/he/subtitle.txt]: English copy for Hebrew locale

**Review by AliSoftware — APPROVED** [review]
- [x] Version bump in `.xcconfig` files
 - [x] ~Latest `.strings` app translations pulled from Glotpress~
    - As you noted, they were apparently all already pulled during last beta 👍 
 - [x] Latest app and metadata translations pulled from Glotpress
     - I noticed a few inconsistencies, especially Hebrew using English copy.
     - But as you noted in your PR description, you already noticed that not every metadata was AppStore-ready and that you will loop back with Team Global on them.
     - Ultimately I think that even if we don't get the latest updated metadata in all locale this time around, this should not block the finalization and publication of 19.4 to end users: that is probably not that much a big deal if it takes one more sprint to get the new metadata properly translated and fixed only in 19.5

**Pull request merged**